packages/table/src/CellDefinitions/ActionCell.tsx

```diff
@@ -1,30 +1,100 @@
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import * as Popper from 'popper.js';
+import { useWindowDimensions } from '@availity/hooks';
+
 import Icon from '@availity/icon';
-import TableActionMenu from '../TableActionMenu';
+import TableActionMenu, { TableActionMenuProps } from '../TableActionMenu';
 import TableActionMenuItem from '../TableActionMenuItem';
 import { Cell, IdType } from '../types/ReactTable';
 import { RecordAction } from '../types/RecordAction';
 import { PrimaryRecordAction } from '../types/PrimaryRecordAction';
+import { UncontrolledTooltip, UncontrolledTooltipProps } from 'reactstrap';
 
 export interface ActionCellConfig<T> {
   actions: RecordAction<T>[];
   primaryAction?: PrimaryRecordAction<T>;
+  tableActionMenuProps?: Omit<TableActionMenuProps, 'children'>;
+  tooltipProps?: UncontrolledTooltipProps;
+  isSticky?: boolean;
 }
 
 const ActionCell = <T extends IdType>({
   actions,
   primaryAction,
+  tableActionMenuProps,
+  tooltipProps,
+  isSticky,
 }: ActionCellConfig<T>): ((cell: Cell<T>) => JSX.Element) => {
   const ActionCellDef = ({ row: { original, id } }: Cell<T>): JSX.Element => {
+    const { height, width } = useWindowDimensions();
+
+    const [menuPositionTransform, setMenuPositionTransform] = useState<string | undefined>();
+
+    const stickyMenuModifiers = useMemo(
```

Comment:
We have a requirement (albeit weird) where when scrolling, if the action menu is open it needs to maintain the same position on window resize and scrolling. 

Adding this optional popper modifier to be able to track and and maintain the initial position, as popper calculates the position dynamically and out of the box will reposition.